### PR TITLE
[Code LLama] add section on vscode extension

### PR DESCRIPTION
**Pull request by mishig25**
Add section about [vscode extension](https://marketplace.visualstudio.com/items?itemName=HuggingFace.huggingface-vscode) supporting Code Llama

### CODE REVIEW

File: codellama.md [hunk 1]
```diff
@@ -317,6 +319,12 @@ You can try out Text Generation Inference on your own infrastructure, or you can
 
 You can learn more on how to [Deploy LLMs with Hugging Face Inference Endpoints in our blog](https://huggingface.co/blog/inference-endpoints-llm). The [blog](https://huggingface.co/blog/inference-endpoints-llm) includes information about supported hyperparameters and how to stream your response using Python and Javascript.
 
+### Using VS Code extension
+
+[HF Code Autocomplete](https://marketplace.visualstudio.com/items?itemName=HuggingFace.huggingface-vscode) is a VS Code extension for testing open source code completion models. The extension was developed as part of [StarCode project](/blog/starcoder#tools--demos), and was updated to support [codellama/CodeLlama-13b-hf](/codellama/CodeLlama-13b-hf). Find more [here](https://github.com/huggingface/huggingface-vscode#code-llama) on how to install and run the extension with Code Llama.
```
Comment (osanseviero) [inline codellama.md:324]:
```suggestion
[HF Code Autocomplete](https://marketplace.visualstudio.com/items?itemName=HuggingFace.huggingface-vscode) is a VS Code extension for testing open source code completion models. The extension was developed as part of [StarCoder project](/blog/starcoder#tools--demos) and was updated to support the medium-sized base model, [Code Llama 13B](https://huggingface.co/codellama/CodeLlama-13b-hf). Find more [here](https://github.com/huggingface/huggingface-vscode#code-llama) on how to install and run the extension with Code Llama. 
```

**Review by osanseviero — APPROVED** [review]
Very nice!

File: codellama.md [hunk 2]
```diff
@@ -317,6 +319,12 @@ You can try out Text Generation Inference on your own infrastructure, or you can
 
 You can learn more on how to [Deploy LLMs with Hugging Face Inference Endpoints in our blog](https://huggingface.co/blog/inference-endpoints-llm). The [blog](https://huggingface.co/blog/inference-endpoints-llm) includes information about supported hyperparameters and how to stream your response using Python and Javascript.
 
+### Using VS Code extension
+
+[HF Code Autocomplete](https://marketplace.visualstudio.com/items?itemName=HuggingFace.huggingface-vscode) is a VS Code extension for testing open source code completion models. The extension was developed as part of [StarCoder project](/blog/starcoder#tools--demos) and was updated to support the medium-sized base model, [Code Llama 13B](https://huggingface.co/codellama/CodeLlama-13b-hf). Find more [here](https://github.com/huggingface/huggingface-vscode#code-llama) on how to install and run the extension with Code Llama.
```
Comment (mishig25) [inline codellama.md:324]:
```suggestion
[HF Code Autocomplete](https://marketplace.visualstudio.com/items?itemName=HuggingFace.huggingface-vscode) is a VS Code extension for testing open source code completion models. The extension was developed as part of [StarCoder project](/blog/starcoder#tools--demos) and was updated to support the medium-sized base model, [Code Llama 13B](/codellama/CodeLlama-13b-hf). Find more [here](https://github.com/huggingface/huggingface-vscode#code-llama) on how to install and run the extension with Code Llama. 
```

**Review by pcuenca — APPROVED** [review]
Awesome!

**Pull request merged**